core/src/data_sources/data_source.rs

```diff
@@ -1192,6 +1246,7 @@ pub async fn cmd_upsert(
             tags,
             source_url,
             text,
+            true, // preserve system tags
```

Comment:
@spolu wondering what the best behaviour is for CMD. I would be tempted to return / overwrite all tags in CMD as it's really a different use-case. WDYT ?

Reply:
no worries about `cmd` we will kill it with fire soon whatever works

Reply:
Ok 👌